src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx

```diff
@@ -494,55 +483,59 @@ const UploadFileUI = ({
-                            />
-                          )}
-                        </div>
-                        <h3>{file.name.substring(0, 16) + "..."}</h3>
-                        <span>
-                          {moment(file.lastModifiedDate).format(
-                            "MMM DD, YYYY, h:mm:ss A"
-                          )}
-                        </span>
-                      </Card>
-                    );
-                  })
-                : ""}
+              <div className={styles.openFolder}>
+                {!toggleFileView
+                  ? getFileListFromFolderID.map((file, i) => {
+                      return (
+                        <Card className={styles.files} key={i}>
+                          <div className={styles.dropdown_menu}>
+                            <Dropdown
+                              overlay={menu_files(file.id, file)}
+                              trigger={["click"]}
+                            >
+                              <div
+                                className="ant-dropdown-link"
+                                onClick={(e) => e.preventDefault()}
+                              >
+                                <MoreOutlined className="Menu_Iconss" />
+                              </div>
+                            </Dropdown>
+                          </div>
+                          <div className={styles.imageFiles}>
+                            {file.thumbUrl ? (
+                              <Image
+                                alt="Image"
+                                src={file.thumbUrl}
+                                height={"150px"}
+                                width={"140px"}
+                              />
+                            ) : file.type == "text/plain" ? (
+                              <FileTextFilled
+                                className={styles.FileTextTwoOneClass}
+                              />
+                            ) : file.type == "application/pdf" ? (
+                              <FilePdfFilled
+                                className={styles.FilePdfTwoToneColor}
+                              />
+                            ) : (
+                              <FileUnknownFilled
+                                className={styles.FileTextTwoOneClass}
+                              />
+                            )}
+                          </div>
+                          <h3>{file.file_name.substring(0, 16) + "..."}</h3>
+                          <span>
+                            {moment
+                              .unix(file.created_at)
+                              .format("MMM DD, YYYY, h:mm:ss A")}
+                          </span>
+                        </Card>
+                      );
+                    })
+                  : ""}
+              </div>
             </div>
-          ) : (
+          ) : !openFolder ? (
```

Comment:
As discussed over skype, please refactor this.

Reply:
done